Framework/DataHandling/src/SaveDiffFittingAscii.cpp

```diff
@@ -274,11 +276,11 @@ std::map<std::string, std::string> SaveDiffFittingAscii::validateInputs() {
   std::string bankNumber = getPropertyValue("Bank");
   std::vector<std::string> splitBank = splitList(bankNumber);
 
-  if (runNumber.empty()) {
-    errors["Bank"] = "Please provide a valid bank list";
-  }
-  if (splitBank.empty()) {
-    errors["Bank"] = "Please provide a valid bank list";
+  if (bankNumber.empty()) {
```

Comment:
This cross-check of parameters belongs in a [`validateInputs`](https://github.com/mantidproject/mantid/blob/master/Framework/API/inc/MantidAPI/IAlgorithm.h#L113) method. That way it is run *before* the `exec` starts and can annotate the invalid parameters in the algorithm dialog.